nearmint/src/lib.rs

```diff
@@ -365,18 +370,21 @@ mod tests {
         let req_commit = RequestCommit::new();
         nearmint.commit(&req_commit);
 
-        let result = nearmint.view_account(&"alice.near".to_string()).unwrap();
-        assert_eq!(result.amount, 999999999990);
-        let result = nearmint.view_account(&"bob.near".to_string()).unwrap();
-        assert_eq!(result.amount, 1000000000010);
+        let alice_info = nearmint.view_account(&"alice.near".to_string()).unwrap();
+        // Should be strictly less, because the rent was applied too.
+        assert!(alice_info.amount < TESTING_INIT_BALANCE - money_to_send);
+        let bob_info = nearmint.view_account(&"bob.near".to_string()).unwrap();
+        // The balance was applied but the rent was not subtracted because we have not performed
+        // interactions from that account.
+        assert_eq!(bob_info.amount, TESTING_INIT_BALANCE + money_to_send);
```

Comment:
should we charge rent on receipt receival as well? because we are updating account any way there.

Reply:
Will do once I clean up mana and gas.